Helm/opensearch-dashboards/values.yaml

```diff
@@ -0,0 +1,136 @@
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image: "opensearchproject/opensearch-dashboards"
+imageTag: "latest"
+imagePullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+
+rbac:
+  create: true
+
+
+podAnnotations: {}
+
+extraEnvs:
+  - name: "NODE_OPTIONS"
+    value: "--max-old-space-size=1800"
+
+podSecurityContext: {}
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
+
+
+config:
+  dashboards.yml:
+    server:
+        name: dashboards
+        host: 0.0.0.0
+
+    ## Dashboards TLS Config
+        ssl:
+          enabled: true
+          key: /usr/share/dashboards/certs/dashboards-key.pem
+          certificate: /usr/share/dashboards/certs/dashboards-crt.pem
+    opensearch.ssl.certificateAuthorities: /usr/share/dashboards/certs/dashboards-root-ca.pem
+
+priorityClassName: ""
+
+httpPort: 5601
+
+opensearchAccount:
+  secret: ""
+  keyPassphrase:
+    enabled: false
+
+
+labels: {}
+
+hostAliases: []
+#- ip: "127.0.0.1"
+#  hostnames:
+#  - "foo.local"
+#  - "bar.local"
+
+serverHost: "0.0.0.0"
+
+service:
+  type: ClusterIP
+  loadBalancerIP: ""
+  port: 5601
```

Comment:
Should be just use the `httpPort`?

Reply:
Reason?

Reply:
DRY. We should use one or the other. Otherwise people have to set it twice.

Reply:
👍